sdk/communication/azure-communication-phonenumbers/test/_shared/utils.py

```diff
@@ -32,3 +32,13 @@ def get_http_logging_policy(**kwargs):
         }
     )
     return http_logging_policy
+
+def get_header_policy(**kwargs):
+    header_policy = HeadersPolicy(**kwargs)
+
+    import os
```

Comment:
Is there a reason why this import is not at the top of the file?

Reply:
I took as example the others functions on this file and them import when needed, however I'm not sure if the current structure is the by product of a refactor or was intentional.

Reply:
looking at the file as a whole makes sense to me to refactor this to have the imports on the top. 

Any inputs on this @whisper6284 and @lucasrsant ?

Reply:
Looking at the other functions I can notice a couple of things:

- The functions are dynamically importing local packages only;
- The functions are dynamically importing packages because they *either* return one or the other class instance.

Since that's not the case of the `os` package, I don't think there is a reason to do so.